submissions/oluwabusolamiEluyela/html-forms/contact.html

```diff
@@ -24,6 +29,7 @@ <h1>Contact Form</h1>
         <textarea rows="17" cols="50" placeholder="Please type your message here"></textarea><br><br>
         <button>Submit</button>
     </fieldset>
+</section>
 </form>
```

Comment:
Why did you use a form closing tag here? I can see an opening tag for form anywhere in your code.

Reply:
Resolved, Please review